src/Silverback.Integration/Diagnostics/IntegrationLogEvents.cs

```diff
@@ -522,6 +523,24 @@ public static class IntegrationLogEvents
             GetEventId(78, nameof(ErrorProcessingOutbox)),
             "Error occurred processing the outbox.");
 
+        /// <summary>
+        ///     Gets the <see cref="LogEvent" /> representing the log that is written when an invalid message
+        ///     has been produced (see <see cref="MessageValidationMode"/>).
+        /// </summary>
+        public static LogEvent InvalidMessageProduced { get; } = new(
+            LogLevel.Warning,
+            GetEventId(79, nameof(InvalidMessageProduced)),
+            "An invalid message has been produced. | validation errors: {validationErrors}");
```

Comment:
Now that I accurately looked across the different files I see that the message formatting is not consistent. Since you concatenate the errors using a new line it will look like this:
```
An invalid message has been produced. | validation errors: FirstName is too long
Email doesn't match the regex
Age cannot be a negative number
```

I would prefer if we could make it look like:
```
An invalid message has been produced. | validation errors: {validationErrors}:
- FirstName is too long
- Email doesn't match the regex
- Age cannot be a negative number
```

What do you think?

Reply:
👍 I adjusted it.